framework/FormControlPanel/FormControlPanel.bbj

```diff
@@ -41,6 +41,7 @@ class public FormControlPanel
 
     method public void setResultSet(ResultSet rs!)
         #RS! = rs!
+        #dataRow! = new DataRow(rs!)
```

Comment:
what does that mean - build a DataRow object from a ResultSet?  Create a DataRow from a ResultSet? 
Take the first record? What if the ResultSet is null() or empty? 
It's unclear to me what is the dataRow! variable versus the ResultSet!in this class, maybe a few comments would do goo.